mono/metadata/gc-internals.h

```diff
@@ -437,12 +439,16 @@ void mono_gc_register_altstack (gpointer stack, gint32 stack_size, gpointer alts
 
 gboolean mono_gc_is_critical_method (MonoMethod *method);
 
+G_BEGIN_DECLS // FIXMEcxx THREAD_INFO_TYPE varies between declaration and definition
```

Comment:
This can be removed if e.g. https://github.com/mono/mono/pull/10297 is accepted.